reactnative/README.md

```diff
@@ -0,0 +1,31 @@
+#React Native Implementation
+
+Visit the [React Native page](http://propertycross.com/reactnative/) on the PropertyCross website for screenshots.
+
+##Introduction
+
+[React Native](https://facebook.github.io/react-native/) enables you to build world-class application experiences on native platforms using a consistent developer experience based on JavaScript and React. The focus of React Native is on developer efficiency across all the platforms you care about — learn once, write anywhere. Facebook uses React Native in multiple production apps and will continue investing in React Native.
+
+##Building the Application
+
+###iOS
+
+To produce an iOS build you will need to have access to a Mac with the latest version of XCode installed. The only setup the application needs is for an 'npm install' to be ran. Once this command has been ran a build can be produced by running the application within XCode.
+
+###Android
+
+To run this application on an Android device you will need to have the Android SDK installed. The Android SDK can be downloaded from http://developer.android.com/sdk/installing/index.html?pkg=tools. In the SDK manager you will need to install ‘Android SDK Tools’, ‘Android SDK Platform Tools’, ‘Android SDK Build-tools’ and ‘Android SDK Platform 23’. The next step is to create the Android project files by running the 'react-native android' command in the project's main directory and to download the project's dependencies which can be done by running the 'npm install' command.
+
+To produce a build for the project you need to run the project's build.gradle. One way of doing this is by running it within [Android Studio](http://developer.android.com/sdk/installing/index.html?pkg=studio). You can import the project into Android Studio using the build.gradle file. Once the build.gradle has been ran it will produce apk files in the android/app/build/outputs/apk folder. These can be uploaded to a device or virtual device by using the adb install <apk-path> command.
```

Comment:
The build.gradle file is created within the 'Android' folder when you run the 'react-native android' command. I'll make this more clear in the README.